Data structures/Dictionary keys/task.md

```diff
@@ -2,6 +2,6 @@
 
 Try to figure out what's wrong with this dictionary and fix it.
 
-<div class="hint">Remember that dictionary keys can only be immutable types? You could replace the list with several individual key:value pairs with 
-keys of type <code>str</code> or with a tuple.</div>
-<div class="hint">Remember that keys need to be unique? You could rename one of the 'Bob' keys.</div>
\ No newline at end of file
+When fixing, be aware of the following requirements
+- Since the dictionary keys can be only immutable types, replace the list with several individual `key:value` pairs with keys of type `str` or with a tuple.
```

Comment:
Since dictionary keys can only be of immutable types, replace the list with several individual 'key:value' pairs with keys of type 'str' or with a tuple.